docs/data/sql_functions_zh.yml

```diff
@@ -640,7 +640,7 @@ temporal:
       3 代表` TO_TIMESTAMP_LTZ(epochMilliseconds, 3)`。
   - sql: TO_TIMESTAMP(string1[, string2])
     table: toTimestamp(STRING1[, STRING2])
-    description: 将 'UTC+0' 时区下格式为 string2（默认为：'yyyy-MM-dd HH:mm:ss'）的字符串 string1 转换为时间戳。
+    description: 将格式为 string2（默认为：'yyyy-MM-dd HH:mm:ss'）的字符串 string1 转换为 timestamp，与时区无关。
```

Comment:
I'm not sure if the Chinese translation matches with my suggestion for the English translation, but if not, please fix accordingly.